tests/test_gcsuri.py

```diff
@@ -276,6 +277,36 @@ def test_gcsuri_read(gcs_v6_txt):
     assert u.read(byte=True) == v6_txt_contents().encode()
 
 
+def test_gcsuri_find_all_files_and_rmdir(gcs_test_path):
+    """Test two methods:
+        - find_all_files()
+        - rmdir()
+
+    Make a directory structure with empty files.
+
+    Check if find_all_files() returns correct file (not sub-directory) paths.
+    Check if rmdir() deletes all empty files on given $prefix.
+    """
+    prefix = os.path.join(gcs_test_path, 'test_gcsuri_find_all_files_and_rmdir')
+    all_files = make_files_in_dir(prefix, make_local_empty_dir_d_a=False)
+
+    # test find_all_files()
+    all_files_found = GCSURI(prefix).find_all_files()
+    assert sorted(all_files_found) == sorted(all_files)
+    for file in all_files:
+        assert GCSURI(file).exists
+
+    # test rmdir(dry_run=True)
+    AutoURI(prefix).rmdir(dry_run=True)
+    for file in all_files:
+        assert GCSURI(file).exists
+
+    # test rmdir(dry_run=False)
+    AutoURI(prefix).rmdir(dry_run=False)
+    for file in all_files:
+        assert not GCSURI(file).exists
```

Comment:
Would prefer to see this separated into three test functions.

Reply:
See the above comment.